demo-project/demo-project-integration-test/docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '3.1'
+
+services:
+  app:
+    image: logb/demo-project:0.0.1-SNAPSHOT
```

Comment:
Good enough to just say `integration-test` here as a fixed string tag for the docker image.